test/units/TEST-58-REPART.sh

```diff
@@ -1459,6 +1466,20 @@ testcase_compression() {
 
     # TODO: add btrfs once btrfs-progs v6.11 is available in distributions.
     for format in squashfs erofs; do
+        if [[ "$format" == erofs && "${ID_LIKE:-}" == alpine && "$OFFLINE" == no ]]; then
+            # erofs in alpine/postmarketos seems to be broken. Maybe, it is built without linux/fs.h and
+            # ioctl(BLKGETSIZE/BLKGETSIZE64) is skipped. Hence, systemd-repart fails with the following:
```

Comment:
Did you report this to PMOS? Sounds like a bug that should get fixed?

Reply:
https://gitlab.alpinelinux.org/alpine/aports/-/issues/17766

Reply:
Fixed https://gitlab.alpinelinux.org/alpine/aports/-/commit/297d4935cc710e5a4b3fadf17d7befeb6a71825b